tests/integration/bond_test.py

```diff
@@ -403,3 +406,77 @@ def test_change_bond_option_miimon(bond99_with_2_slaves):
     bond_options["miimon"] = "200"
     libnmstate.apply(desired_state)
     assertlib.assert_state(desired_state)
+
+
+def test_create_bond_without_mode():
```

Comment:
Do we need to check that `fail_over_active` is only specified together with`active-failover`? Could `fail_over_mac=active` be in the current state (but ignored) and then mode:active-failover be in the desired state?

Reply:
The new patch just check on merged interface state, so should be no difference with existing.
If you insist, I can create one.